src/gepetto_viewer_rerun/client.py

```diff
@@ -74,27 +74,93 @@ def createScene(self, sceneName: str):
         )
         logger.info(msg)
 
-    def _getSceneIndex(self, sceneName: str):
-        for index, scene in enumerate(self.sceneList):
+    def getWindowList(self):
+        return [scene.name for scene in self.windowList]
+
+    def getSceneList(self):
+        return [scene.name for scene in self.sceneList]
+
+    def getWindowID(self, name: str):
+        assert isinstance(name, str), "Parameter 'name' must be a string"
+
+        if name not in self.getWindowList():
+            logging.error(f"getWindowID(): Unknown windowName '{name}'.")
+            return ""
+        logger.info("getWindowID() only checks if window exists and return its name.")
```

Comment:
maybe this is supposed to provide an index number. 
@florent-lamiraux : are you fine with this method returning a string, or would you expect an int ?

Reply:
In fact method `getWindowId` is deprecated. It returns the string given as input. I suggest that we remove this method.